app/views/repairs/index.html.erb

```diff
@@ -1,5 +1,5 @@
-| <%= link_to "New Repair", new_repair_path(@repairs) %> |
-<% @repairs.each do |repair| %>
+| <%= link_to "New Repair", new_repair_path(@repair) %> |
+<% @repair.each do |repair| %>
```

Comment:
On style -- you need to be aware of when you are dealing with one entry or many, and your variable names should reflect this.  In this case, you want to use repairs as the variable name, not repair, because you are dealing with a bunch of them